lib/remote_send.js

```diff
@@ -343,9 +339,6 @@ var File = exports.File = function(name, port, host, reconnect, on_connect_func)
             // it's opened, hooray
             this.mark_as_up.bind(this)();
 
-            // Store the fd for later use.
-            this._fd = fd;
```

Comment:
This looks risky - I don't have the code top of mind anymore, but this looks like somethign we rely on being there.

Reply:
I only see it assigned, but never used: https://github.com/jib/piped/search?utf8=%E2%9C%93&q=_fd&type=Code

Reply:
I'll trust you on this, but my experience with node (at least older versions) is that it's incredibly finicky with filedescriptors and they're a PITA to debug, so please tread carefully.

Reply:
Ack